library/std/src/primitive_docs.rs

```diff
@@ -555,6 +555,7 @@ impl Copy for () {
 ///
 /// ```
 /// # #![feature(rustc_private)]
+/// #[allow(unused_extern_crates)]
```

Comment:
This doesn't feel like a reasonable fix to me? In particular if `extern crate libc;` *is* unused, shouldn't it just be removed?

Reply:
It's not used in `library/std/src/primitive_docs` but used in `library/core/src/primitive_docs`.

So I have to put this on `primitive_docs` under both `core` and `std` to overcome tidy and doctest errors.